sorting/strand_sort.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <list>
+
+template <typename T>
+std::list<T> strand_sort(std::list<T> lst) {
```

Comment:
Missing function documentation, parameter
documentation and return statement documentation.

See #962 as a reference.